lib/browser/api/menu.js

```diff
@@ -45,30 +45,49 @@ Menu.prototype._init = function () {
 
 Menu.prototype.popup = function (window, x, y, positioningItem) {
   let [newX, newY, newPosition, newWindow] = [x, y, positioningItem, window]
+  let opts
 
   // menu.popup(x, y, positioningItem)
-  if (!window) {
-    // shift over values
-    if (typeof window !== 'object' || window.constructor !== BrowserWindow) {
-      [newPosition, newY, newX, newWindow] = [y, x, window, null]
-    }
+  if (window && typeof window !== 'object' || window.constructor !== BrowserWindow) {
```

Comment:
> `if (window && ...`

I guess it's a mistake.
This check was supposed to test that `null` of `undefined` were not explicitly provided as a `window`, right?
But what if it's a call in this form:
```js
menu.popup(/* x */ 0, /* y */ 0, ...)
```
`0` won't pass this check and the shift won't happen.

Let's change the check to `if (window != null) && ...`